libs/scraper-agent/mocked/mockAgent.ts

```
@@ -6,21 +6,23 @@ import type { NamedProcessor } from "../src/processors";
 import { SectionType, sections } from "../src/processors/types";
 import type { IAskUser, SectionName, EventSection } from "../src/types";
 import { sleep } from "@thecointech/async/sleep";
+import { AccountResponse } from "@thecointech/vqa";
```

ESLint

[error] `@thecointech/vqa` import should occur before type import of `../src/agent`

(import/order)